internal/controller/monovertexrollout/monovertexrollout_progressive.go

```diff
@@ -0,0 +1,35 @@
+package monovertexrollout
+
+import (
+	"context"
+
+	"github.com/numaproj/numaplane/internal/controller/progressive"
+	"github.com/numaproj/numaplane/internal/util/kubernetes"
+	"github.com/numaproj/numaplane/internal/util/logger"
+	apiv1 "github.com/numaproj/numaplane/pkg/apis/numaplane/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// TODO: can move other progressive interface functions here as well? (same with PipelineRollout, ISBServiceRollout)
```

Comment:
I'll do this in a follow up PR - didn't want to make this harder to read by doing it in this one